distro_data.json

```diff
@@ -351,6 +323,26 @@
             "stdc++": "5.3.1"
         }
     },
+    "openSUSE Leap 42.3": {
+        "release": {
+            "date": "2017-07",
+            "eol": "2019-012"
```

Comment:
Noticed the typo here just now. Is it actually 01 or 12?

Reply:
01 according to https://en.opensuse.org/Lifetime

Reply:
confirmed, sorry for the typo